frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -297,7 +297,10 @@ object Interpreter {
                   val classpath = project.fullClasspath(dag, state.client)
                   val coursierClasspathArgs =
                     classpath.flatMap(elem => Seq("--extra-jars", elem.syntax))
-                  val ammoniteCmd = (coursierCmd ++ coursierClasspathArgs).mkString(" ")
+
+                  val ammArgs = Seq("-- --no-home-predef")
```

Comment:
Would it make sense to forward the arguments from the users instead?

```scala
bloop console foo -- --no-home-predef
```

What should users do who don't want `--no-home-predef`?

Reply:
Yes you are right. Though I would say that not using the home predef should be the default but maybe that is just me. I will see if I can figure this out.